### PR TITLE
add HackerRank 

### PR DESCRIPTION
**Pull request by NMW03**
This PR adds [HackerRank ](https://www.hackerrank.com/ )presence
Screenshots:
Main
Dashboard
Challenges
Challenge
Skills Verification

### CODE REVIEW

File: websites/H/HackerRank/presence.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const presence = new Presence({
+  clientId: "655534149871992845"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: Math.floor(Date.now() / 1000)
+  },
```
Comment (Bas950) [inline websites/H/HackerRank/presence.ts:9]: you should probs set your Date.now outside of the updatadata event

**Review by Bas950 — APPROVED** [review]
but the rest looks good

File: websites/H/HackerRank/presence.ts (hunk 1 above)
Comment (NMW03) [reply websites/H/HackerRank/presence.ts:9]: why 🤔

Comment (Bas950) [reply websites/H/HackerRank/presence.ts:9]: Because you will set a new time now every updateData event, instead of just keep setting the same.

**Review by ririxidev — APPROVED** [review]

File: websites/H/HackerRank/presence.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const presence = new Presence({
+  clientId: "655534149871992845"
+}),
+  startTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp
```
Comment (Bas950) [inline websites/H/HackerRank/presence.ts:9]: yea that wont work lmao you would need
`startTimestamp: startTimestamp`

**Review by Bas950 — CHANGES_REQUESTED** [review]
but the rest looks good

**Review by Bas950 — APPROVED** [review]

**Pull request merged**